azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest.java

```diff
@@ -69,9 +77,11 @@ public void setUp() throws Exception {
     jobtypeManager =
         new JobTypeManager(null, null, this.getClass().getClassLoader());
     JobTypePluginSet pluginSet = jobtypeManager.getJobTypePluginSet();
-    pluginSet.addPluginClass("java", JavaJob.class);
+    pluginSet.setCommonPluginLoadProps(AllJobExecutorTests.setUpCommonProps());
     pluginSet.addPluginClass("test", InteractiveTestJob.class);
     fakeProjectLoader = new MockProjectLoader(workingDir);
+    Utils.initServiceProvider();
+    AzkabanExecutorServer.setApp(new FakeApp());
```

Comment:
How does FlowRunner depend on AzkabanExecutorServer? In the future, we should refactor FlowRunner so that lower layer components shouldn't depend on higher level components.